.github/workflows/check-scripts-updates.yml

```diff
@@ -20,8 +20,8 @@ jobs:
 
         changed_files=$(git diff --name-only "$BASE_COMMIT" "$HEAD_COMMIT")
 
-        # Check if any files in scripts/ were updated (excluding wrap_scripts_in_yaml.py)
-        scripts_updated=$(echo "$changed_files" | grep -v 'wrap_scripts_in_yaml.py' | grep 'scripts/' || true)
```

Comment:
The file should be removed because it is surpassed by `misc/sync_scripts.py`? I still see it present in repo

Reply:
Yea, I forgot to remove it, thanks. Removed.